frontend/src/utils/testUtils.ts

```diff
@@ -30,45 +31,47 @@ export const fillInput = async (
     }
   }
 
-  await wait(() => {
```

Comment:
No need to use **wait** here anymore. The new version of react-testing-library wraps all calls to **fireEvent** with wait

Reply:
Liking these changes!